spec-main/api-tray-spec.ts

```diff
@@ -12,17 +13,26 @@ describe('tray module', () => {
     tray = null as any
   })
 
-  describe('tray.setContextMenu', () => {
-    afterEach(() => {
-      tray.destroy()
+  ifdescribe(process.platform === 'darwin')('tray get/set ignoreDoubleClickEvents', () => {
+    it('returns false by default', () => {
+      const ignored = tray.getIgnoreDoubleClickEvents()
+      expect(ignored).to.be.false('ignored')
     })
 
-    it('accepts menu instance', () => {
-      tray.setContextMenu(new Menu())
+    it('can be set to true', () => {
+      tray.setIgnoreDoubleClickEvents(true)
+
+      const ignored = tray.getIgnoreDoubleClickEvents()
+      expect(ignored).to.be.true('not ignored')
     })
+  })
 
-    it('accepts null', () => {
-      tray.setContextMenu(null)
+  describe('tray.setContextMenu(menu)', () => {
+    it('accepts both null and Menu as parameters', () => {
+      expect(() => { tray.setContextMenu(new Menu()) }).to.not.throw()
+      expect(() => { tray.setContextMenu(null) }).to.not.throw()
+
+      tray.destroy()
```

Comment:
is it ok for this to not be called if one of the `expect`s above fails? if so, maybe we can just not call it at all?